lms/djangoapps/django_comment_client/forum/tests.py

```diff
@@ -342,8 +342,8 @@ class SingleThreadQueryCountTestCase(ModuleStoreTestCase):
 
     @ddt.data(
         # old mongo with cache
-        (ModuleStoreEnum.Type.mongo, 1, 6, 4, 17, 8),
-        (ModuleStoreEnum.Type.mongo, 50, 6, 4, 17, 8),
+        (ModuleStoreEnum.Type.mongo, 1, 6, 4, 18, 8),
+        (ModuleStoreEnum.Type.mongo, 50, 6, 4, 18, 8),
         # split mongo: 3 queries, regardless of thread response size.
         (ModuleStoreEnum.Type.split, 1, 3, 3, 17, 8),
```

Comment:
I tried stepping through the test-- the query to get the disabled xblocks is done in both cases. But split uses a bulk operations context manager, correct? That could be why there is not an increase in number.

````
with self.bulk_operations(CourseLocator(org, course, run)):
            item = super(DraftVersioningModuleStore, self).create_course(
                org, course, run, user_id, master_branch=master_branch, **kwargs
            )